src/Makefile.am

```diff
@@ -27,7 +27,7 @@ scrconf_DATA = \
   scrconf/drbd_lvm.scr
 
 desktop_DATA = \
-  desktop/drbd.desktop
+  desktop/org.openSUSE.YaST.DRBD.desktop
```

Comment:
Real file is "desktop/org.opensuse.yast.DRBD.desktop", should use the lower case